src/components/ha-assist-chat.ts

```diff
@@ -40,7 +41,11 @@ export class HaAssistChat extends LitElement {
 
   @query("#message-input") private _messageInput!: HaTextField;
 
-  @query("#scroll-container") private _scrollContainer!: HTMLDivElement;
+  @query(".message:last-child")
+  private _lastChatMessage!: LitElement;
```

Comment:
The `!` here seems optimistic as there won't be a message when first opening the chat

Reply:
I think you're right. Looks like it is populated early in `willUpdate` so the message is there before the first render.
It's weird that the conversation is reset to `[]` in `disconnectedCallback` but there is no `connectedCallback` to repopulate it. It probably isn't needed right now but then `this._conversation = []` in `disconnectedCallback` also shouldn't be needed.